sqlglot/dialects/clickhouse.py

```diff
@@ -158,6 +159,7 @@ class Parser(parser.Parser):
                 this=seq_get(args, 2), expression=seq_get(args, 1), unit=seq_get(args, 0)
             ),
             "DATE_FORMAT": _build_date_format,
+            "DATE_TRUNC": date_trunc_to_time,
```

Comment:
Why do we need this?

Reply:
Uh, I wanted to add a test case for generating date_trunc with the zone parameter.

I wanted to use CH -> CH to test it, and found that clickhouse cannot parse date_trunc.

Then I added this parser...

Reply:
I just tried parsing it with CH and found no issue:

```python
>>> import sqlglot
>>> sqlglot.parse_one("DATE_TRUNC('HOUR', a, 'UTC')", "clickhouse")
DateTrunc(
  unit=Literal(this=HOUR, is_string=True),
  this=Column(
    this=Identifier(this=a, quoted=False)),
  zone=Literal(this=UTC, is_string=True))
>>> sqlglot.parse_one("DATE_TRUNC('HOUR', a, 'UTC')", "clickhouse").sql("clickhouse")
"DATE_TRUNC('HOUR', a, 'UTC')"
```

Reply:
I remembered it wrong, and I didn't check it before coding😭